src/commands/guess.ts

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable max-len */
+import dayjs from 'dayjs';
+import { CommandInteraction, MessageActionRow, MessageButton, PermissionString } from 'discord.js';
+import { RESTPostAPIChatInputApplicationCommandsJSONBody } from 'discord-api-types/v10';
+import { Command } from '../interfaces/Command';
+import { isUserInDB, registerUser, getDay, getWord, setPlayed } from '../database';
+import { runAtMidnight } from '../utils/runner';
+import { awaitMessage } from '../utils/msgCollector';
+import { isValid } from '../utils/checkWord';
+import { toDefault, toEmoji } from '../utils/converters';
+import { plataform } from '../utils/plataform';
+import { square } from '../utils/emotes.json';
+
+let usersTries: Record<string, { id: string, attempts: string[] }> = {};
+let activeGames: string[] = [];
+
+runAtMidnight(() => {
+	usersTries = {};
+	activeGames = [];
+});
+
+export default class Guess implements Command {
+	commandStructure: RESTPostAPIChatInputApplicationCommandsJSONBody = {
+		'name': 'adivinhar',
+		'description': 'Tente adivinhar a palavra do dia!',
+	};
+
+	dev = true;
+
+	permissions: PermissionString[] = ['MANAGE_MESSAGES'];
+
+	async execute(interaction: CommandInteraction) {
+		const { user, channel } = interaction;
+
+		if (await isUserInDB(user.id)) {
+			const t = dayjs().tz('America/Sao_Paulo').endOf('day').unix();
+			interaction.reply({
+				'content': `Você já jogou hoje!\nTempo restante até a próxima palavra: <t:${t}:R>`,
+				'ephemeral': true,
+			});
+
+			return;
+		} else {
+			registerUser(user.id);
+		}
+
+
+		const gameMessage: Record<number, string> = Object.create({});
```

Comment:
```suggestion
		const gameMessage: Record<number, string> = {};
```

(I think they're equivalent)

Reply:
Second refator on it LOL